lib/lib.scriptHost.d.ts

```diff
@@ -1,294 +0,0 @@
-/*! *****************************************************************************

```

Comment:
That's `scripthost.d.ts` versus `scriptHost.d.ts` — might cause trouble on Unix.